include/seqan3/alphabet/nucleotide/all.hpp

```diff
@@ -55,104 +55,82 @@
- * |   K      | G *or* T               |      ☑         |      ☐       |      ☐       |      ☑         |      ☐       |      ☐       |
- * |   V      | A *or* C *or* G        |      ☑         |      ☐       |      ☐       |      ☑         |      ☐       |      ☐       |
- * |   H      | A *or* C *or* T        |      ☑         |      ☐       |      ☐       |      ☑         |      ☐       |      ☐       |
- * |   D      | A *or* G *or* T        |      ☑         |      ☐       |      ☐       |      ☑         |      ☐       |      ☐       |
- * |   B      | C *or* G *or* T        |      ☑         |      ☐       |      ☐       |      ☑         |      ☐       |      ☐       |
- * |   N      | A *or* C *or* G *or* T |      ☑         |      ☑       |      ☐       |      ☑         |      ☑       |      ☐       |
- * | **Size** |                        |     15         |      5       |      4       |     15         |      5       |      4       |
+ * | Letter   | Description            |                   seqan3::dna15      |                   seqan3::dna5       |                  seqan3::dna4        |                seqan3::rna15         |                    seqan3::rna5      |                 seqan3::rna4         |
+ * |:--------:|------------------------|:------------------------------------:|:------------------------------------:|:------------------------------------:|:------------------------------------:|:------------------------------------:|:------------------------------------:|
+ * |   A      | Adenine                |                              A       |                              A       |                              A       |                              A       |                              A       |                              A       |
+ * |   C      | Cytosine               |                              C       |                              C       |                              C       |                              C       |                              C       |                              C       |
+ * |   G      | Guanine                |                              G       |                              G       |                              G       |                              G       |                              G       |                              G       |
+ * |   T      | Thymine (DNA)          |                              T       |                              T       |                              T       | <div style="color:LightGrey">U</div> | <div style="color:LightGrey">U</div> | <div style="color:LightGrey">U</div> |
+ * |   U      | Uracil (RNA)           | <div style="color:LightGrey">T</div> | <div style="color:LightGrey">T</div> | <div style="color:LightGrey">T</div> |                              U       |                              U       |                              U       |
```

Comment:
> use <span ...> instead of <div...>, because you most likely don't want block elements in a table cell.

I have no clue what you are talking about, I just copy'n'pasted this. If span is better I can use that.

Reply:
xD :+1: